packages/gatsby/src/utils/worker/child/queries.ts

```diff
@@ -0,0 +1,44 @@
+import { setState } from "./state"
+import {
+  IGroupedQueryIds,
+  runPageQueries,
+  runStaticQueries,
+} from "../../../services"
+import { store } from "../../../redux"
+import { GraphQLRunner } from "../../../query/graphql-runner"
+import { getDataStore } from "../../../datastore"
+import { buildSchema } from "./schema"
+
+export function setQueries(): void {
+  setState([`components`, `staticQueryComponents`])
+}
+
+export async function runQueries(queryIds: IGroupedQueryIds): Promise<void> {
+  const workerStore = store.getState()
+
+  // If buildSchema() didn't run yet, execute it
+  if (workerStore.schemaCustomization.composer === null) {
```

Comment:
Compared different states of this and after successful schema building this was always defined